sources/platform/integrations/programming/webhooks/events.md

```diff
@@ -1,34 +1,33 @@
 ---
-title: Events
-description: Specify the types of events that trigger a webhook in an Actor or task run. Trigger an action on Actor or task run creation, success or failure.
+title: Events types for webhooks
+description: Specify the types of events  that trigger a webhook in an Actor or task run. Trigger an action on Actor or task run creation, success, failure, abortion, or timeout.
 sidebar_position: 1
 slug: /integrations/webhooks/events
 ---
 
-# Events
 
-**Specify the types of events that trigger a webhook in an Actor or task run. Trigger an action on Actor or task run creation, success or failure.**
+**Specify the types of events  that trigger a webhook in an Actor or task run. Trigger an action on Actor or task run creation, success, failure, terminationo timeout.**
```

Comment:
Nit: `termination or timeout`. Not sure if we want to call it termination or abortion 🤷